src/marketplace/MakeATankSaleView.js

```diff
@@ -4,6 +4,8 @@ import Tank from '../tanks/Tank.js';
 import {getAllUsersTanks} from '../globalComponents/tankAPIIntegration.js';
 import {getUser} from '../globalComponents/userAPIIntegration.js';
 import {makeASale} from './marketPlaceAPIConnections.js';
+import { ToastContainer ,toast } from 'react-toastify';
```

Comment:
if you are going to copy and paste something, make sure the original version was right